src/test/java/com/atlassian/bitbucket/jenkins/internal/util/TestUtils.java

```diff
@@ -0,0 +1,34 @@
+package com.atlassian.bitbucket.jenkins.internal.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.nio.file.Paths;
+
+import static java.nio.file.Files.readAllBytes;
+import static org.apache.commons.io.Charsets.UTF_8;
+
+public class TestUtils {
+
+    public static final String BITBUCKET_BASE_URL = "http://localhost:7990/bitbucket";
+    public static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+    public static String readFileToString(String relativeFilename) {
+        try {
+            return new String(
+                    readAllBytes(Paths.get(TestUtils.class.getResource(relativeFilename).toURI())));
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static String encode(String urlSnippet) {
+        try {
+            return URLEncoder.encode(urlSnippet, UTF_8.toString());
```

Comment:
```suggestion
            return URLEncoder.encode(urlSnippet, StandardCharsets.UTF_8.name());
```
and remove the apache.commons import